bitnami/dataplatform-bp2/Chart.yaml

```diff
@@ -0,0 +1,61 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 1.0.0
+dependencies:
+  - condition: kafka.enabled
+    name: kafka
+    repository: https://charts.bitnami.com/bitnami
+    version: 12.x.x
+  - condition: spark.enabled
+    name: spark
+    repository: https://charts.bitnami.com/bitnami
+    version: 5.x.x
+  - condition: elasticsearch.enabled
+    name: elasticsearch
+    repository: https://charts.bitnami.com/bitnami
+    version: 14.x.x
+  - condition: logstash.enabled
+    name: logstash
+    repository: https://charts.bitnami.com/bitnami
+    version: 3.x.x
+  - condition: wavefront.enabled
+    name: wavefront
+    repository: https://charts.bitnami.com/bitnami
+    version: 2.x.x
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 1.x.x
+description: OCTO Data platform Kafka-Spark-Elasticsearch Helm Chart
+engine: gotpl
+home: https://github.com/bitnami/charts/tree/master/bitnami/dataplatform-bp2
+icon: https://bitnami.com/assets/stacks/dataplatform-bp2/img/dataplatform-bp2-stack-220x234.png
+keywords:
+  - dataplatform
+  - kafka
+  - spark
+  - elasticsearch
+  - logstash
+  - kibana
+  - zookeeper
+  - apache
+  - tanzuobservability
+  - wavefront
+maintainers:
+  - email: containers@bitnami.com
+    name: Bitnami
```

Comment:
Should the OCTO team be listed here?

Reply:
mmm not sure, they didn't add themselves in bp1 neither